spec/fixtures/course/programming_messages_test_report.xml

```diff
@@ -0,0 +1,463 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+	<testsuite errors="0" failures="2" name="L4Q11_Public" skipped="0" tests="2" time="2.008">
+		<testcase classname="L4Q11_Public" name="test_public_fail_test" time="0.000">
+			<meta expected="6188" expression="num_combination(17, 5)" hint="" output="-1"/>
+			<failure message="-1 != 6188 : Wrong answer" type="AssertionError">
+<![CDATA[Traceback (most recent call last):
+  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/timeout_decorator/timeout_decorator.py", line 69, in new_function
+    return function(*args, **kwargs)
+  File "answer.py", line 65, in test_public_fail_test
+    self.assertEqual(num_combination(17, 5), 6188, 'Wrong answer')
+AssertionError: -1 != 6188 : Wrong answer
+]]>			</failure>
+		</testcase>
+		<testcase classname="L4Q11_Public" name="test_public_timeout_error" time="2.008">
+			<meta expected="True" expression="recursion" hint="Is your function written recursively?"/>
+			<failure message="'Timed Out'" type="TimeoutError">
+<![CDATA[Traceback (most recent call last):
+  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/timeout_decorator/timeout_decorator.py", line 69, in new_function
+    return function(*args, **kwargs)
```

Comment:
@fonglh Can we shorten the report and  make it more readable and modifiable ? 

The report in the evaluator is quite standard: https://github.com/Coursemology/evaluator-slave/blob/master/spec/fixtures/sample_report.xml